drivers/wifi/simplelink/simplelink_sockets.c

```diff
@@ -679,13 +679,6 @@ static int map_credentials(int sd, const void *optval, socklen_t optlen)
 }
```

Comment:
The commit "use compat option", pleas change the commit subject to
`drivers: wifi: simplelink: Use socket option from socket.h`
Also all commits need to have some description in the commit body, there is now none in this commit.